README.md

```diff
@@ -3,8 +3,10 @@
 
 Register [Marathon](https://mesosphere.github.io/marathon/) Tasks as [Consul](https://www.consul.io/) Services for service discovery.
 
-`marathon-consul` takes information provided by the [Marathon event bus](https://mesosphere.github.io/marathon/docs/event-bus.html) and
-forwards it to Consul agents. It also re-syncs all the information from Marathon
+`marathon-consul` takes information provided by the [Marathon event tream](https://mesosphere.github.io/marathon/docs/event-bus.html) which is implementing W3 [Server Site Events](http://www.w3schools.com/html/html5_serversentevents.asp) and
```

Comment:
I wrote a note about future callback deprecation and separated section about SSE support, rest of documentation left unchanged